common/consts/consts.go

```diff
@@ -0,0 +1,8 @@
+package consts
+
+// EthHappyPathFinalizationDepth is the number of blocks that must be included on top of a block for it to be considered "final",
+// under happy-path aka normal network conditions.
+//
+// See https://docs.optimism.io/stack/transactions/transaction-finality for a quick TLDR explanation,
```

Comment:
feel like this is a counter-intuitive explanation since it refers to the lifecycle of a rollup tx. Prefer we use more L1 agnostic terminology like [commitment levels](https://www.alchemy.com/overviews/ethereum-commitment-levels)

Reply:
nice link. updated: [f255bc8](https://github.com/Layr-Labs/eigenda-proxy/pull/241/commits/f255bc8679b056cec4d9086cf47601dfe9a40abd)